### ISSUE TITLE
Key details display does not match key properties

### ISSUE THREAD

**Issue by apex-hughin** [mailvelope/mailvelope#250]
I created a key using Mailvelope for Chrome version 0.11.0 and exported it.
I imported it in GnuPG and noticed that there are two keys: The primary key is a [CS] RSA 2048 bit key, the subkey is an [E] RSA 2047 bit key.
In “Display Keys” both keys are shown as 2048 bit RSA (Encrypt or Sign) keys.

[CS] certify, sign
[E] encrypt

**Comment by Grannath** [comment]
I seem to have the same issue. I have created a key pair on the command line using GPG on Windows. I created a main key with a subkey, one E and the other CS, both RSA 4096 bit. But Mailvelope shows me both of them as E and CS.

I sincerely hope this is only a display problem. It should be fixed nonetheless. Using separate keys for signing and encrypting is recommended, so it should be fully supported by Mailvelope.
